src/nvim/ex_docmd.c

```diff
@@ -7357,6 +7357,12 @@ static void ex_terminal(exarg_T *eap)
   do_cmdline_cmd(ex_cmd);
 }
 
+/// ":fclose"
+static void ex_floatclose(exarg_T *eap)
```

Comment:
Seems more conventional to name the `ex_` function using the same name as the command: `ex_fclose`

Reply:
before think about this. because C has `fclose`